python/ray/_private/services.py

```diff
@@ -548,23 +548,23 @@ def create_redis_client(redis_address, password=None):
     """
     if not hasattr(create_redis_client, "instances"):
         create_redis_client.instances = {}
-    else:
-        cli = create_redis_client.instances.get(redis_address)
-        if cli is not None:
-            try:
-                cli.ping()
-                return cli
-            except Exception:
-                create_redis_client.instances.pop(redis_address)
 
-    redis_ip_address, redis_port = extract_ip_port(
-        canonicalize_bootstrap_address(redis_address))
-    # For this command to work, some other client (on the same machine
-    # as Redis) must have run "CONFIG SET protected-mode no".
-    create_redis_client.instances[redis_address] = redis.StrictRedis(
-        host=redis_ip_address, port=int(redis_port), password=password)
+    for _ in range(10):
```

Comment:
name the 10 to avoid magic number? i think maybe better to put it in some file and all redis related retry can be controlled there.

Reply:
I will use `START_REDIS_WAIT_RETRIES` here.